sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/ReactorNettyClient.java

```diff
@@ -197,11 +197,7 @@ public HttpHeaders headers() {
         @Override
         public Flux<ByteBuf> body() {
             return bodyIntern()
-                .doOnSubscribe(this::updateSubscriptionState)
-                .map(byteBuf -> {
-                    byteBuf.retain();
-                    return byteBuf;
-                });
+                .doOnSubscribe(this::updateSubscriptionState);
```

Comment:
what this change for ?

Reply:
Just removed the unnecessary `map` operator. We don't need to retain the byteBuf here, as it is not released afterwards. 
This can potentially raise IllegalReferenceCount issue with netty. 
Although, we don't use this API, which is why no one has seen this issue.